### DISCUSSION TITLE
Dicussions are open now

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [jupyter/nbgrader#1630] [General]
Thanks for closing #1627.
There are a few issues labeled as https://github.com/jupyter/nbgrader/labels/discussion that could be moved here now by applying the
"Converting issues to discussion" procedure as outlined in https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions.

**Comment by jhamrick** [comment]
Thanks, I've converted the existing discussion issues to discussions.
